lib/charliecloud.py

```diff
@@ -217,6 +217,13 @@ def copy_unpacked(self, other):
       DEBUG("copying image: %s -> %s" % (other.unpack_path, self.unpack_path))
       copytree(other.unpack_path, self.unpack_path, symlinks=True)
 
+   def delete (self):
+      if(unpacked_image_p(self.unpack_path)):
```

Comment:
```suggestion
   def unpack_delete(self):
      if (unpacked_image_p(self.unpack_path)):
```

(you'll also need to re-alphabetize it)